<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>670</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>670</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>1100</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Импорт результатов тестов на COVID-19 в ЕМИАС</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="main_top_frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>450</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="path_to_table_big_frame">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>140</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QFrame" name="path_to_table_frame">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>30</y>
            <width>594</width>
            <height>91</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame#path_to_table_frame {
	border-color: rgb(194, 233, 233);
	border-width: 1px;
	border-style: double;

}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QPushButton" name="path_to_table_button">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>131</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI Semibold</family>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	background-color: rgb(38, 153, 153);
	color: rgb(240,240,240);
	border-style: none;
}
QPushButton:hover{
	background-color: rgb(30, 134, 134);
}
QPushButton:pressed{
	background-color: rgb(22, 114, 114);
}</string>
           </property>
           <property name="text">
            <string>Выбрать файл...</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="path_to_table_input">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>30</y>
             <width>291</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit{
	border-color: rgb(153, 153, 153);
}
QLineEdit:hover{
	border-color: rgb(117, 117, 117);
}
QLineEdit:focus{
	border-color: rgb(61, 207, 207);
}
</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>  Выберите файл, нажав на кнопку слева</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QFrame" name="covid_frame">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>450</x>
             <y>10</y>
             <width>141</width>
             <height>71</height>
            </rect>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="covid_descript">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">Qlabel {
	background-color: rgba(255,255,255,0);
	border-color: rgba(255,255,255,0);
}</string>
              </property>
              <property name="text">
               <string>COVID-статус</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="path_to_table_ok">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QRadioButton{
	color:rgb(0, 135, 0)
}
QRadioButton:disabled{
	color:rgb(150,150,150)
}</string>
              </property>
              <property name="text">
               <string>Отрицательный</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="path_to_table_ill">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QRadioButton{
	color: rgb(135, 0, 0)
}
QRadioButton:disabled{
	color:rgb(150,150,150)
}</string>
              </property>
              <property name="text">
               <string>Положительный</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QFrame" name="path_to_table_deco">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>116</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame#path_to_table_deco {
	background-color: rgb(240, 240, 240)
	
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="path_to_table_descript">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>121</width>
             <height>20</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">Qlabel {
	background-color: rgba(255,255,255,0);
	border-color: rgba(255,255,255,0);
}</string>
           </property>
           <property name="text">
            <string>Путь до таблицы</string>
           </property>
          </widget>
         </widget>
         <widget class="QToolButton" name="toolButton">
          <property name="geometry">
           <rect>
            <x>500</x>
            <y>0</y>
            <width>101</width>
            <height>23</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton {
	background-color: rgb(38, 153, 153);
	color: rgb(240,240,240);
	border-style: none;
}
QToolButton:hover{
	background-color: rgb(30, 134, 134);
}
QToolButton:pressed{
	background-color: rgb(22, 114, 114);
}</string>
          </property>
          <property name="text">
           <string>Настройки</string>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextOnly</enum>
          </property>
         </widget>
         <widget class="QToolButton" name="toolButton_2">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>0</y>
            <width>200</width>
            <height>23</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton {
	background-color: rgb(200, 75, 75);
	color: rgb(240,240,240);
	border-style: none;
}
QToolButton:hover{
	background-color: rgb(180, 65, 65);
}
QToolButton:pressed{
	background-color: rgb(160, 55, 55);
}</string>
          </property>
          <property name="text">
           <string>Сообщить об ошибке в программе</string>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextOnly</enum>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>18</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>19</number>
          </property>
          <item>
           <widget class="QPushButton" name="start_parsing_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI Semibold</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: rgb(38, 153, 153);
	color: rgb(240,240,240);
	border-style: none;
}
QPushButton:hover{
	background-color: rgb(30, 134, 134);
}
QPushButton:pressed{
	background-color: rgb(22, 114, 114);
}
QPushButton:disabled{
	background-color: rgb(180,180,180);
}</string>
            </property>
            <property name="text">
             <string>Обработать данные из таблицы</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="login_big_frame">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QFrame" name="login_frame">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>20</y>
            <width>594</width>
            <height>191</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame#login_frame {
	border-color: rgb(194, 233, 233);
	border-width: 1px;
	border-style: double;

}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="username_descript">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>141</width>
             <height>20</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">Qlabel {
	background-color: rgba(255,255,255,0);
	border-color: rgba(255,255,255,0);
}</string>
           </property>
           <property name="text">
            <string>Имя пользователя</string>
           </property>
          </widget>
          <widget class="QLabel" name="password_descript">
           <property name="geometry">
            <rect>
             <x>308</x>
             <y>30</y>
             <width>61</width>
             <height>20</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">Qlabel {
	background-color: rgba(255,255,255,0);
	border-color: rgba(255,255,255,0);
}</string>
           </property>
           <property name="text">
            <string>Пароль</string>
           </property>
          </widget>
          <widget class="QPushButton" name="start_main_algorithm">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>9</x>
             <y>130</y>
             <width>575</width>
             <height>40</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI Semibold</family>
             <pointsize>13</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	background-color: rgb(38, 153, 153);
	color: rgb(240,240,240);
	border-style: none;
}
QPushButton:hover{
	background-color: rgb(30, 134, 134);
}
QPushButton:pressed{
	background-color: rgb(22, 114, 114);
}
QPushButton:disabled{
	background-color: rgb(180,180,180);
}</string>
           </property>
           <property name="text">
            <string>Перенести данные из таблицы в систему</string>
           </property>
          </widget>
          <widget class="QSplitter" name="splitter">
           <property name="geometry">
            <rect>
             <x>9</x>
             <y>55</y>
             <width>575</width>
             <height>31</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="handleWidth">
            <number>20</number>
           </property>
           <widget class="QLineEdit" name="username_input">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{
	border-color: rgb(153, 153, 153);
}
QLineEdit:hover{
	border-color: rgb(117, 117, 117);
}
QLineEdit:focus{
	border-color: rgb(61, 207, 207);
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="placeholderText">
             <string>  Введите имя пользователя</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
           <widget class="QLineEdit" name="password_input">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{
	border-color: rgb(153, 153, 153);
}
QLineEdit:hover{
	border-color: rgb(117, 117, 117);
}
QLineEdit:focus{
	border-color: rgb(61, 207, 207);
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="placeholderText">
             <string>  Введите пароль</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </widget>
         </widget>
         <widget class="QFrame" name="login_frame_deco">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>104</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame#login_frame_deco {
	background-color: rgb(240, 240, 240)
	
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="login_frame_descript">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>121</width>
             <height>20</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">Qlabel {
	background-color: rgba(255,255,255,0);
	border-color: rgba(255,255,255,0);
}</string>
           </property>
           <property name="text">
            <string>Вход в систему</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="table_parse_error_descript">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>18</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="table_widget">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="autoScrollMargin">
       <number>8</number>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Корректность</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Извлеченные данные</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="wtrm">
      <property name="font">
       <font>
        <family>Segoe UI</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	color:rgb(100,100,100)
}</string>
      </property>
      <property name="text">
       <string>Figurin Ivan, 2020</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="indent">
       <number>3</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
